README.md

```diff
@@ -86,6 +88,35 @@ $ npm config set script-shell "C:\\Program Files\\Git\\bin\\bash.exe"
 
 You must have [Git for Windows](https://git-scm.com/download/win) installed. See this [StackOverflow answer](https://stackoverflow.com/a/46006249) for more details.
 
+### Build the bot database
+
+
+*As we use prisma for managing our DB it is up to you what relational database framework you use.*
+
+[The following is a guide to setting up Postgres inside a docker container.](https://github.com/docker-library/docs/blob/master/postgres/README.md) While the choice of database is up to you, the instructions for getting started in this guide assume you are using a postgres docker image. 
+After you have Postgres (or your DB of choice) up and running edit the line in your .env file to be 
+```
+DATABASE_URL=postgres://pg_user:pg_pass@pg_host:pg_port/pg_table
+# required for any DB functionality, we will get this URL in a later section
+```
+- pg_user  = The Username you set in your POSTGRES_USER environment variable (default postgres)
+- pg_pass  = The Password you set in your POSTGRES_PASS environment variable (default postgres)
+- ph_host  = The IP of the server running your Postgres instance (default localhost)
+- pg_port  = The Port assigned to your Postgres instance (default 5432)
+- pg_table = The Name of the table you wish to use for the bot
+
+
+The first time you run this project you can run the following command to initialize the database
+```
+$ npm run prisma:generate
```

Comment:
The `prisma:generate` script only generates types at dev time. `npm ci` or `npm install` do the same thing, since Prisma also generates types during client install.

I think you mean `npm run baseline`, which ~~is~~ will be set up to run `prisma migrate resolve` with a set of tables with which to initialize the database, as a... baseline.

```suggestion
$ npm run baseline
```